dev/TeachingTip/TeachingTip.cpp

```diff
@@ -958,8 +958,12 @@ void TeachingTip::OnHeroContentPlacementChanged()
 
 void TeachingTip::OnF6AcceleratorKeyClicked(const winrt::CoreDispatcher&, const winrt::AcceleratorKeyEventArgs& args)
 {
-    if (args.VirtualKey() == winrt::VirtualKey::F6)
+    if (!args.Handled() &&
+        IsOpen() &&
+        args.VirtualKey() == winrt::VirtualKey::F6 &&
+        (args.EventType() == winrt::CoreAcceleratorKeyEventType::KeyDown || args.EventType() == winrt::CoreAcceleratorKeyEventType::SystemKeyDown))
     {
+        args.Handled(true);
         //  Logging usage telemetry
```

Comment:
Setting this to true always may be a bit aggressive - consider setting this to true only when you really take an action, like when you call .Focus.

Better yet, you can set it to true only if Focus returns true, in case the thing you try to focus can't be focused.